### DISCUSSION TITLE
Road access for Large Temples?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#519] [Q&A] [answered]
I have a curiosity regarding how the road access is required in Augustus for Large Temples.

![Screenshot_20211109-161832047](https://user-images.githubusercontent.com/93785920/140940818-c19e56f6-129c-4b70-9812-69ac5a744a63.jpg)
Given this temple (placed originally in Julius), it is connected to a road on the North corner. Workers are still assigned, but it is telling me that this temple, as well as others, are not operational.

I understand that Grand Temples need the road access strictly on the one middle tile on SE/SW, but does the same apply for large temples? Does the road need to hit strictly the one tile with stairs?

**Comment by Ouaz** [comment, marked as answer]
> I understand that Grand Temples need the road access strictly on the one middle tile on SE/SW, but does the same apply for large temples? Does the road need to hit strictly the one tile with stairs?

Large temples has to be connected on the middle tile on any side (during construction and after).

- Not connected in Augustus (but it is in Julius)
![not_connected](https://user-images.githubusercontent.com/11946570/140967564-35eb06b0-f37b-41ae-afd0-a4ef7559e561.png)

- Connected in Augustus (on middle tile)
![connected](https://user-images.githubusercontent.com/11946570/140967662-719ea1c2-d996-4d31-920a-c87a9312cff2.png) ![not_connected2](https://user-images.githubusercontent.com/11946570/140966942-00f61b4a-a9c3-4ce3-ba9f-e638b64f9eaf.png)

**Reply by catalin-duta** [reply]
Bummer. Guess I'll have to put something else in corners.

Thanks for the reply!

**Reply by crudelios** [reply]
This is a bug though, I think. Only during construction should they require connection to the middle tile. @Keriew am I correct?
